login.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://uiplugin.netlify.app/src.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="https://uiplugin.netlify.app/src.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>User Login</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="header-1">
+        <a href="index.html" class="logo">
+          <i class="fa-solid fa-bag-shopping"></i> StyleVilla
+        </a>
+        <form action="" class="search-form">
+          <input
+            type="search"
+            name=""
+            placeholder="What are you looking for?"
+            id="search-box"
+          />
+          <label for="search-box" class="fas fa-search"></label>
+        </form>
+
+        <div class="icons">
+          <div id="search-btn" class="fas fa-search"></div>
+          <a href="login.html" class="fas fa-user"></a>
+          <a href="product.html" class="fa-solid fa-shirt"></a>
+          <a href="wishlist.html" class="fas fa-heart"></a>
+          <a href="cart.html" class="fas fa-shopping-cart"></a>
+        </div>
+      </div>
+    </header>
+
+    <!-- login form -->
+    <form class="form-container">
+      <h1 class="title">Login</h1>
+      <div class="input">
+        <label>Email</label>
+        <input type="input" placeholder="Email" />
+      </div>
+
+      <div class="input">
+        <label>Password</label>
+        <input type="password" placeholder="Password" />
+      </div>
+
+      <div class="input-remember">
+        <label
+          ><input type="checkbox" /> Remeber me? <br /><br />
+          <a href="#">Forgot your Password?</a></label
+        >
+      </div>
+
+      <div class="hero-btns">
+        <button class="btn">Login</button>
+      </div>
+      <br />
+      <div class="create">
+        <a href="signup.html">Create New Account ></a>
```

Comment:
here you can give some descriptive class name. Don't use common/single names for classes.

Reply:
done